gst/gst.go.h

```diff
@@ -1,6 +1,7 @@
 #ifndef __GST_GO_H__
 #define __GST_GO_H__
 
+#include <stdlib.h>
```

Comment:
I'm personally okay with this here, I just want to double check that it doesn't mess with anything on my setup later when I'm off work. 

I'm pretty sure it would have an include guard anywhere it's implemented though because the C standard says they should be safe to include more than once.